### PR TITLE
add cmd test for kubectl auth can-i

### PR DESCRIPTION
**Pull request by CaoShuFeng**
**Release note**:

```
NONE
```

### CODE REVIEW

**Comment by eparis** [comment]
@k8s-bot ok to test
/approve
jordan, david anything more we should test or anything wrong with these?

File: hack/make-rules/test-cmd-util.sh [hunk 1]
```diff
@@ -3694,6 +3695,21 @@ __EOF__
     rm "${SAR_RESULT_FILE}"
   fi
 
+  # kubectl auth can-i
+  # kube-apiserver is started with authorization mode AlwaysAllow, so kubectl can-i always returns yes
```
Comment (liggitt) [inline hack/make-rules/test-cmd-util.sh]: not sure how meaningful testing an authorization check is if the answer will always be yes

Comment (CaoShuFeng) [inline hack/make-rules/test-cmd-util.sh]: I can add some abnormal test cases when this is merged:
https://github.com/kubernetes/kubernetes/pull/46432

Comment (CaoShuFeng) [inline hack/make-rules/test-cmd-util.sh]: @liggitt 
Some more tests are added.

**Comment by CaoShuFeng** [comment]
@k8s-bot pull-kubernetes-e2e-kops-aws test this

File: hack/make-rules/test-cmd-util.sh (hunk 1 above)
Comment (CaoShuFeng) [inline hack/make-rules/test-cmd-util.sh]: ping

**Comment by CaoShuFeng** [comment]
ping

**Comment by deads2k** [comment]
/lgtm

others have tried, but maybe I'm the special one?

**Comment by deads2k** [comment]
> /lgtm
> 
> others have tried, but maybe I'm the special one?

@liggitt behold my power! :)

**Comment by liggitt** [comment]
🙇

**Pull request merged**